bigtable/filter.go

```diff
@@ -125,6 +127,15 @@ func (cf columnFilter) proto() *btpb.RowFilter {
 // See https://github.com/google/re2/wiki/Syntax for the accepted syntax.
 func ValueFilter(pattern string) Filter { return valueFilter(pattern) }
 
+// ValueFilterInt64 returns a filter that matches cells whose value
+// matches the provided RE2 pattern.
+// It performs the 64-bit big-endian signed integer encoding.
+func ValueFilterInt64(pattern int) Filter {
```

Comment:
Same `int64` argument and `binary.Write` comments here and for `ValueRangeFilterInt64`.